7.0/Dockerfile

```diff
@@ -32,10 +39,12 @@ RUN apt-get update && apt-get install -y \
   php-memcached \
   php-mysql \
   php-soap \
-  php-xml \
-  unzip \
-  zip
+  php-xml
```

Comment:
When I have used this same Dockerfile for other projects, sometimes I have gotten feedback from the docker instance that php-mbstring.

Reply:
Nevermind.  I see this in the code below.